File: 프로그래머스/unrated/181942. 문자열 섞기/문자열 섞기.cpp
#include <string>
#include <vector>

using namespace std;

string solution(string str1, string str2) {
    string answer = "";
    int cnt = str1.size() + str2.size();
    for (int i = 0; i < cnt; i++){
        int a = i/2;
        if (i%2 == 0)
            answer += str1[a];
        else if (i%2 == 1)
            answer += str2[a];
    }
    return answer;
}

File: 프로그래머스/unrated/181942. 문자열 섞기/문자열 섞기.swift
import Foundation

func solution(_ str1:String, _ str2:String) -> String {
    let arr1 = Array(str1)
    let arr2 = Array(str2)
    var answer = [Character]()
    for i in 0..<arr1.count {
        answer.append(arr1[i])
        answer.append(arr2[i])
    }
    return String(answer)
}